### PR TITLE
1768 - update event catalog instructions and information

### PR DESCRIPTION
**Pull request by christineyost**
* Update CODE VA documentation on `consume-events` page to provide more information about fields in the `catalog-info.yaml` file.
* Update Event Catalog documentation on `produce-events` page to match current Event Entity schema.
* Small update to the `use-events` page to remove a duplicate point

### CODE REVIEW

File: docs/consume-events.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ See for instance this Java code that consumes messages from a topic named “tes
 
 ### Register with CODE VA
 
-[CODE VA](https://code.va.gov/) (formerly known as the Hub) is a software catalog that houses entities from across VA. Once your consumer application is up and running, register with the catalog so event producers are aware of how their events are being used, and which systems are consuming them.
+[CODE VA](https://code.va.gov/) (must be on VA network to view) is a software catalog that houses information about software entities from across the VA. Once your consumer application is up and running, it's important to register with the catalog so event producers are aware of how their events are being used and which systems are consuming them.
```
Comment (jamielwhite) [inline docs/consume-events.md:163]: FYI the [style guide](https://design.va.gov/content-style-guide/word-list#v) says not to use "the" in front of VA as a noun

```suggestion
[CODE VA](https://code.va.gov/) (must be on VA network to view) is a software catalog that houses information about software entities from across VA. Once your consumer application is up and running, it's important to register with the catalog so event producers are aware of how their events are being used and which systems are consuming them.
```

File: docs/produce-events.md [hunk 2]
```diff
@@ -165,11 +165,11 @@ See for instance this Java code that produces messages to a topic named “test
 
 ### Register with CODE VA
 
-[CODE VA](https://code.va.gov/) (formerly known as the Hub) is a software catalog that houses entities from across VA. Once you have your producer application up and running, it's important to register with the catalog to ensure that both current and future consumers can discover your event and access its details.
+[CODE VA](https://code.va.gov/) (must be on VA network to view) is a software catalog that houses information about software entities from across the VA. Once you have your producer application up and running, it's important to register with the catalog to ensure that both current and future consumers can discover your event and access its details.
```
Comment (jamielwhite) [inline docs/produce-events.md:168]:
```suggestion
[CODE VA](https://code.va.gov/) (must be on VA network to view) is a software catalog that houses information about software entities from across VA. Once you have your producer application up and running, it's important to register with the catalog to ensure that both current and future consumers can discover your event and access its details.
```

File: docs/consume-events.md (hunk 1 above)
Comment (christineyost) [reply docs/consume-events.md:163]: Ah, good to know!

**Review by jamielwhite — APPROVED** [review]

File: docs/consume-events.md [hunk 3]
```diff
@@ -160,55 +160,89 @@ See for instance this Java code that consumes messages from a topic named “tes
...
+1. In CODE VA, an event-consuming software entity can be modeled as a [Component](https://backstage.io/docs/features/software-catalog/descriptor-format/#kind-component) or as a [System](https://backstage.io/docs/features/software-catalog/descriptor-format#kind-system). If you are unsure whether to classify your consumer as a Component or a System, see the [Backstage System Model](https://backstage.io/docs/features/software-catalog/system-model/).
 
+2. Create a file named `catalog-info.yaml` at the root of your source code repository and populate it with the applicable template, updating `metadata` and `spec` with values that correspond to your component or system.
+
+3. Once your `catalog-info.yaml` file has been committed it will be automatically picked up after some time and the software entity will be viewable on [CODE VA](https://code.va.gov/) (must be on the VA network to view). If you would like the event to display quicker, log into [CODE VA](https://code.va.gov/) while on the VA network and follow the [default Backstage provided method](https://backstage.io/docs/features/software-catalog/#adding-components-to-the-catalog) for adding entries to the catalog.
```
Comment (alwerner) [inline docs/consume-events.md:171]: Can we be more precise than "after some time"?

File: docs/consume-events.md [hunk 4]
```diff
@@ -160,55 +160,89 @@ See for instance this Java code that consumes messages from a topic named “tes
...
+        * **url** [required]: The external url that will be opened when the link is clicked.
+        * **title** [required]: Display text for the link.
+* **spec** [required]: A structure that contains information about the component. The `spec` structure includes the following properties.
+    * **type** [required]: The component type. Possible values include: `website`, `service`, `library`, etc.
+    * **lifecycle** [required]: The current development status for the component. Possible values include: `experimental`, `production`, `deprecated`, etc.
+    * **owner** [required]: The team that owns the event-consuming component. If this is set to the name of a GitHub team within the VA's GitHub organization, this field will link to a page with details about the team in CODE VA.
+    * **subscribesToEvent** [required]: An array of strings. Each string must match the `metadata.name` value of a producer's `catalog-info.yaml` file. This field is used to relate the component to the events that it consumes. All consumers of an event will be displayed on the Event Details page.
```
Comment (alwerner) [inline docs/consume-events.md:208]: Can we provide more context for the what the Event Details page is? or maybe link to the Events page?

File: docs/consume-events.md [hunk 5]
```diff
@@ -160,55 +160,89 @@ See for instance this Java code that consumes messages from a topic named “tes
...
+    * **links** [optional]: A list of links related to the system. Each link consists of a `url` and a `title`.
+        * **url** [required]: The external url that will be opened when the link is clicked.
+        * **title** [required]: Display text for the link.
+* **spec** [required]: A structure that contains information about the system. The `spec` structure includes the following properties.
+    * **owner** [required]: The team that owns the event-consuming system. If this is set to the name of a GitHub team within the VA's GitHub organization, this field will link to a page with details about the team in CODE VA.
+    * **domain** [optional]: The VA domain in which a particular system exists. Possible values might be: `claims status`, `health`, `appointments`, `benefits`, etc.
+    * **subscribesToEvent** [required]: An array of strings. Each string must match the `metadata.name` value of a producer's `catalog-info.yaml` file. This field is used to relate the system to the events that it consumes. All consumers of an event will be displayed on the Event Details page.
```
Comment (alwerner) [inline docs/consume-events.md:245]: Same comment as above regarding the Event Details page.

File: docs/produce-events.md [hunk 6]
```diff
@@ -165,99 +165,78 @@ See for instance this Java code that produces messages to a topic named “test
...
-
-3. Once your `catalog-info.yaml` file has been committed, log into [CODE VA](https://code.va.gov/) while on the VA network, and follow the [default Backstage provided method](https://backstage.io/docs/features/software-catalog/#adding-components-to-the-catalog) for adding entries to the catalog.
-
+2. Populate the `catalog-info.yaml` file with an `Event` Backstage entity based on the template below.
+3. Once your `catalog-info.yaml` file has been committed it will be automatically picked up after some time and the event will be viewable on [CODE VA](https://code.va.gov/) (must be on the VA network to view). If you would like the event to display quicker, log into [CODE VA](https://code.va.gov/) while on the VA network and follow the [default Backstage provided method](https://backstage.io/docs/features/software-catalog/#adding-components-to-the-catalog) for adding entries to the catalog.
```
Comment (alwerner) [inline docs/produce-events.md:176]: Same comment here regarding using precise language for "after some time."

File: docs/consume-events.md (hunk 4 above)
Comment (christineyost) [reply docs/consume-events.md:208]: We could say something like
```
* **subscribesToEvent** [required]: An array of strings. Each string must match the `metadata.name` value of a producer's `catalog-info.yaml` file. This field is used to relate the system to the events that it consumes and to display the system on each related event's CODE VA catalog entry.
```

We could link to the DLA Event page, but that page doesn't currently have any consumers

File: docs/consume-events.md (hunk 3 above)
Comment (christineyost) [reply docs/consume-events.md:171]: I can follow up with the CODE VA team about this on Monday

File: docs/consume-events.md (hunk 4 above)
Comment (alwerner) [reply docs/consume-events.md:208]: Yeah, I think that's better.

File: docs/use-events.md [hunk 7]
```diff
@@ -10,8 +10,7 @@ The Event Catalog is the single source of truth for all events currently publish
 
 In the Event Catalog, you’ll find the following information for every event:
 
-* A brief overview of the event, including a description, domain, system of record, the topics it’s published on, expected throughput, the current lifecycle phase it’s in, and what other systems may be consuming it. 
-* Data elements: A summary of what each field in an event is.
+* A brief overview of the event, including a description, domain, system of record, the topics it’s published on, expected throughput, the current lifecycle phase it’s in, and what other systems may be consuming it.
```
Comment (alwerner) [inline docs/use-events.md:13]: Could this be bullets?

**Review by alwerner — APPROVED** [review]
Approved with one minor comment.

File: docs/use-events.md (hunk 7 above)
Comment (jimuhh) [reply docs/use-events.md]: I agree with wanting bullets here. Also, when writing bulleted lists for web content, it helps to pay close attention to [parallel structure](https://webapps.towson.edu/ows/parallelism.htm) , hence:

  
**A brief overview of the event, including its:** 

*   description 
*   domain 
*   system of record
*   expected throughput
*   current lifecycle phase 
*   usage in topics
*   connections to consuming systems

**Pull request merged**